geolocation.py

```diff
@@ -56,6 +56,9 @@ def edit_distance_to_world(location):
     # pylint: disable=invalid-name
 
     # special cities and countries
+
+    location = location.title()
```

Comment:
@kdobolyi, can you please put an explanatory note here? I don't know what this title method does.